internal/handlers/common/killcursors.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 FerretDB Inc.
```

Comment:
What do you think about adding a comment or two about the copyright headers in the contributions guide?

There is more than one way to do the copyrights:
1. always use the same date (the first copyright's date)
2. a new file gets the current year's value in the header
3. increase the date annually, in every file (in which case a transient inconsistency is OK as it will be fixed later by the tool)

The Go project used the second (2) approach, so if you see a mismatching year in a new file, it's something that can be reported on the review. It doesn't look like it's the case here  (2021 is used everywhere), but it should be described somewhere, so we never get any confusion like this in the future.

(Or maybe you have a tool that checks the headers that ensures the year 2021, in which case never mind.)

Reply:
We tend not to document things that are enforced by the linter.

We also can't describe something that could be interpreted by someone somewhere as legal advice without saying "THIS IS NOT A LEGAL ADVICE" in all caps and with much more words than "a comment or two".